server/seeds/demoSeed.js

```diff
@@ -0,0 +1,31 @@
+const User = require("../models/User");
+const mongoose = require("mongoose");
+
+const users = [
+  new User({
+    name: process.env.DEMO_USER_NAME,
+    email: process.env.DEMO_USER_EMAIL,
+    password: process.env.DEMO_USER_PASSWORD,
+  })]
```

Comment:
We need to make a new profile too

Reply:
Can just be generic information like "John Doe"

Reply:
Also, no reason for this to be an array. There will only really ever be one demo user account.

Reply:
updated